services/server/src/lib/tutor.ts

```diff
@@ -68,6 +68,8 @@ export async function constructTutorsCache(date: Dayjs): Promise<TutorsCache> {
     }
   );
 
+  const isOnlineMap = await cache.onlineStatus.isOnlineBatch(onboardedTutors.map(t => t.id));
```

Comment:
We can omit this filed from the cache (and update types). Also, don't forget to update the sorting alogrithm used to order tutors.